admin/app/views/solidus_admin/accounts/show.html.erb

```diff
@@ -3,4 +3,5 @@
   class="text-4xl"
 ><a href="#" data-action="hello#greet">Account</a></h1>
 
-<h2 class="text-3xl underline">Your account info</h1>
+<p>Logged in as <%= current_solidus_admin_user.email %></p>
+<p><%= button_to "Log out", solidus_admin_logout_path, method: solidus_admin_logout_method, class: 'underline' %></p>
```

Comment:
This view could be part of a new "account/show" component to be rendered (like "products/index").
Non-blocking for now, I've added a dedicated point in the list of the final things to do before the release:
- [[Admin] Release SolidusAdmin v0.0.1.alpha.1 #5280](https://github.com/solidusio/solidus/issues/5280)
- [[Admin] Refactor "accounts/show" view into a separate component #5292](https://github.com/solidusio/solidus/issues/5292)